src/main/java/ch/njol/skript/expressions/ExprJoinSplit.java

```diff
@@ -1,98 +1,102 @@
-/**

```

Comment:
Line endings for history / viewing the changes properly 😃

Reply:
I don't know why but this is always an issue

Reply:
Because line endings are not consistent in Skript's files : some are LF and some are CRLF, it's quite random